devsetup/scripts/interfaces-setup.sh

```diff
@@ -28,29 +28,29 @@ EOF
 </controller>
 EOF
 
-    sudo virsh detach-device crc --file /tmp/pci1.xml --persistent || true
+    sudo virsh detach-device $INSTANCE_NAME --file /tmp/pci1.xml --persistent || true
     sleep 1
-    sudo virsh detach-device crc --file /tmp/pci2.xml --persistent || true
+    sudo virsh detach-device $INSTANCE_NAME --file /tmp/pci2.xml --persistent || true
     sleep 10
 
-    sudo virsh attach-device crc --file /tmp/pci1.xml --persistent
+    sudo virsh attach-device $INSTANCE_NAME --file /tmp/pci1.xml --persistent
     sleep 1
-    sudo virsh attach-device crc --file /tmp/pci2.xml --persistent
+    sudo virsh attach-device $INSTANCE_NAME --file /tmp/pci2.xml --persistent
     sleep 10
 
-    sudo virsh start crc
+    sudo virsh start $INSTANCE_NAME
     # wait for the VM to be up
     sleep 240
 fi
 
+virsh --connect=qemu:///system net-update default add-last ip-dhcp-host --xml "<host name='$INSTANCE_NAME' ip='$IP_ADDRESS'/>" --config --live
```

Comment:
Looks like this has broken the baremetal job[1]. Let's see with https://github.com/openstack-k8s-operators/install_yamls/pull/627 that seems to fix it locally for me.

[1] https://review.rdoproject.org/zuul/builds?job_name=cifmw-crc-podified-edpm-baremetal&job_name=openstack-baremetal-operator-crc-podified-edpm-baremetal&job_name=install-yamls-crc-podified-edpm-baremetal&skip=0